security/certificate_types_descriptions/service-ca-certificates.adoc

```diff
@@ -47,6 +47,8 @@ Services that use service CA certificates include:
 * cluster-openshift-apiserver-operator
 * cluster-openshift-controller-manager-operator
 * cluster-samples-operator
+* cluster-storage-operator
+* cluster CSI driver operators
```

Comment:
Should we mention "cluster" in that case, it may be more clear to say CSI Driver operators?

Reply:
Done.